### PR TITLE
This fixes grammar and style for Workspace Admin-> Projects

### PR DESCRIPTION
**Pull request by petra-sargent**
Signed-off-by: Petra Sargent <psargent@redhat.com>

### What does this PR do?
This is a dublicate of #610.  IT was easier to create a new PR because of the merge conflict

### What issues does this PR fix or reference?

### CODE REVIEW

**Comment by petra-sargent** [comment]
@rkratky This is the duplicate PR that cover PR #610. Please review and merge. Thanks!

File: src/main/pages/che-6/workspace-admin/projects.adoc [hunk 1]
```diff
@@ -35,28 +36,40 @@ Projects are always associated with a workspace and saved in a workspace configu
...
+.  A project defines a set of attributes. The attributes of a project can be mandatory or optional. Attributes that are optional can be dynamically set at runtime or during configuration.
+.  Sub-projects may have different project types than their parents. Modules may physically exist within the tree structure of the parent. For example, subdirectories exists within the tree structure of the parent.  Also, modules may physically exist outside the tree structure of the parent such as when the parent is a soft link to the module project.
+
+[id="creating-a-sub-projects"]
+== Creating a sub-project
+
+A sub-project is a portion of a project that can have sets of commands run against it where the sub-directory is treated as the root working directory. Sub-projects make it possible to organize a single repository into multiple, independently buildable, and runnable units.
+
+To create a module, right click on a directory in the IDE explorer tree and select *Convert to Project*. You can then execute commands directly against this sub-projects.
```
Comment (kowen-rh) [inline src/main/pages/che-6/workspace-admin/projects.adoc]: right-click

File: src/main/pages/che-6/workspace-admin/projects.adoc [hunk 2]
```diff
@@ -35,28 +36,40 @@ Projects are always associated with a workspace and saved in a workspace configu
...
 
-image::workspaces/projects.png[]
+* Git projects 
+* remotely hosted archives
+* GitHub projects 
+* example projects provided by Eclipse Che 
 
-[id="sub-projects"]
-== Sub-Projects
+Project import tools can be found on the *User Dashboard* when you are creating a new workspace or editing an existing workspace on the IDE. Project import tools can also be found on the *Workspace* menu.
```
Comment (kowen-rh) [inline src/main/pages/che-6/workspace-admin/projects.adoc]: Regarding "on the IDE" -- did you mean "in the IDE" here? Using "on" seems awkward to me, but I may not be understanding the context.

File: src/main/pages/che-6/workspace-admin/projects.adoc [hunk 3]
```diff
@@ -35,28 +36,40 @@ Projects are always associated with a workspace and saved in a workspace configu
  ]
 ----
 
-Once a project is saved into a workspace config, the IDE will check if the project exists on a file system, and if not, import it using `source.location` URL. It happens at IDE initialization stage.
+Once a project is saved into a workspace configuration, the IDE will check if the project exists on a file system.  If the project does not exist on the file sytem, you can import it using the `source.location` URL. This happens during the IDE initialization stage.
```
Comment (kowen-rh) [inline src/main/pages/che-6/workspace-admin/projects.adoc]: You use "a filesystem" here followed by "the filesystem" -- which applies best? I'd use whichever is most accurate in both cases.

File: src/main/pages/che-6/workspace-admin/projects.adoc (hunk 2 above)
Comment (petra-sargent) [reply src/main/pages/che-6/workspace-admin/projects.adoc]: "in the IDE"

File: src/main/pages/che-6/workspace-admin/projects.adoc (hunk 3 above)
Comment (petra-sargent) [reply src/main/pages/che-6/workspace-admin/projects.adoc]: Since the original text has "a", I think that it should be "a"

**Review by rkratky — APPROVED** [review]

**Pull request merged**